src/lib/radon/ccdr/index.ts

```diff
@@ -27,7 +27,7 @@ export type BlockNumber = number | Bytes32;
 /**
  * Base container class for JSON Remote Procedure Calls.
  */ 
-export class JsonRPC {
+export class CrossChainRPC {
```

Comment:
Why do we need to differentiate in the class between `CrossChainRPC` and regular `JsonRPC`

Reply:
Just context-aware from where this class it's intended to be used.